README.md

```diff
@@ -290,6 +293,36 @@ a { color: @color-white; }
 ```scss
 a { color: $color-white; }
 ```
+#### disableFix
+
+Autofixing the rule violation can be enabled or disabled, like:
+
+```js
+// .stylelintrc
+"rules": {
+  // ...
+  "scale-unlimited/declaration-strict-value": ["/color/", {
+    disableFix: false,
+  }],
+  // ...
+}
+```
+
+#### autofixFunc
+
+Autofixing the rule violation can be acheived using a custom function, like:
+
+```js
+// .stylelintrc
+"rules": {
+  // ...
+  "scale-unlimited/declaration-strict-value": ["/color/", {
+    disableFix: true,
+    autofixFunc: function(root, node, context){ /* css AST change code goes here */ }
```

Comment:
@imvetri 
I just realisied we still should pass which type broke a rule, i.e. `validVar`, `validFunc`, `validKeyword`